### PR TITLE
 add option to change label names

### PR DESCRIPTION
**Pull request by AymanHossam**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,8 +65,9 @@ export default class ValidationComponent extends Component {
   // ex:
   // [{ fieldName: "name", messages: ["The field name is required."] }]
   _addError(fieldName, rule, value, isFn) {
+    let name = this.labels[fieldName];
```
Comment (perscrew) [inline index.js:68]:
```suggestion
    const name = this.labels[fieldName];
```

**Review by perscrew — CHANGES_REQUESTED** [review]
1 small comment

**Comment by arochedy** [comment]
Update the readme to explain and add example ?

**Review by perscrew — COMMENTED** [review]
could you rebase with master branch please ? I merged some Prs into master.

**Comment by AymanHossam** [comment]
done

**Review by perscrew — APPROVED** [review]

**Pull request merged**